frontend/public/components/utils/mask-data.jsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { CopyToClipboard } from './index';
+
+export const MaskedData = () => <React.Fragment>
+  <span className="sr-only">Value hidden</span>
+  <span aria-hidden="true">&bull;&bull;&bull;&bull;&bull;</span>
+</React.Fragment>;
+
+export class MaskData extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { showSecret: false };
+    this.toggleSecret = this.toggleSecret.bind(this);
+  }
+
+  toggleSecret() {
+    this.setState({ showSecret: !this.state.showSecret });
+  }
+
+  render() {
+    const { title, value } = this.props;
+    const { showSecret } = this.state;
+    const visibleValue = showSecret ? value : <MaskedData /> ;
+
+    return <React.Fragment>
+      <dt>{title} {value &&
+        <button className="btn btn-link co-mask-data__btn" style={{paddingLeft: '24px'}} type="button" onClick={this.toggleSecret}>
```

Comment:
I can't get `co-mask-data__btn` to reflect in the browser thus the hard-coded `style=...`.

Reply:
I'd rather add a class and we can help you test if necessary. (We should figure out what's wrong in your environment, though.)

Reply:
I would rather add a class as well, but at this point I can't even change an existing scss value.  Here I changed `top` to `10px`, but when I inspect it it still says 0:
![image](https://user-images.githubusercontent.com/12733153/40935276-93b964ce-6805-11e8-9604-295e0cc7fe26.png)
When I click on the link to the .scss file, it does display the updated value:
![image](https://user-images.githubusercontent.com/12733153/40935299-a58a6a90-6805-11e8-8876-34a7d28f163c.png)

I've tried starting/stopping server and dev, `./build.sh`, `./build-frontend.sh`, `yarn run build`, etc..  Very frustrating, I just can't get scss changes/additions to show up in browser :-(